sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ASTNode.scala

```diff
@@ -55,10 +55,16 @@ case class ASTNode(
   }
 
   /** Origin of the ASTNode. */
-  override val origin = Origin(Some(line), Some(positionInLine))
+  override val origin: Origin = Origin(Some(line), Some(positionInLine))
 
   /** Source text. */
```

Comment:
this comment isn't super useful - can you explain a bit more what "source text" is?